pkg/util/scrubber/default_test.go

```diff
@@ -295,6 +295,62 @@ func TestTextStripURLPassword(t *testing.T) {
 		`Connection dropped : ftp://user:********@host:port`)
 }
 
+func TestTextStripLogPassword(t *testing.T) {
```

Comment:
Can you add one case with the SQL connection error, similar to the one reported by support ?

Reply:
Agree that having more test cases testing the password value it self would be useful 😄